tests/test_optimizer/test_fused_adam_kernel.py

```diff
@@ -41,11 +43,9 @@ def torch_adam_update(
     param.addcdiv_(exp_avg, denom, value=-step_size)
 
 
-@clear_cache_before_run()
```

Comment:
you can keep this to save GPU memory.

Reply:
This is not compatible with `@pytest.mark.parametrize`

Reply:
![image](https://github.com/hpcaitech/ColossalAI/assets/23111350/f3ff6092-f6fa-491a-9ca0-f5dc48605bb2)
This leads to 
![image](https://github.com/hpcaitech/ColossalAI/assets/23111350/cb5c792b-9cf7-4e43-b9be-8790cde840c6)

Reply:
Right, it is only compatible with `colossalai.testing.parameterize`.

Reply:
But it seems that `colossalai.testing.parameterize` does not support usage like 
![image](https://github.com/hpcaitech/ColossalAI/assets/23111350/37ba544f-348d-402d-be45-4498a66c4f6b)

Reply:
I see, that's true.